### PR TITLE
Fixes: U4-7683 Search on Property Editors/Data Types in new Content Type Editor should display two results - resuse / available

### PR DESCRIPTION
**Pull request by madsrasmussen**

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/views/common/overlays/contenttypeeditor/editorpicker/editorpicker.html [hunk 1]
```diff
@@ -76,6 +66,47 @@
 
 	</div>
 
+	<!-- FILTER RESULTS -->
+	<div ng-if="vm.showFilterResult">
+
+		<h5 class="-border-bottom -black">Reuse</h5>
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web.UI.Client/src/views/common/overlays/contenttypeeditor/editorpicker/editorpicker.html]: Localize?

File: src/Umbraco.Web.UI.Client/src/views/common/overlays/contenttypeeditor/editorpicker/editorpicker.html [hunk 2]
```diff
@@ -76,6 +66,47 @@
...
+						<a href="" title="{{ dataType.name }}">
+							<i class="{{ dataType.icon }}" ng-class="{'icon-autofill': dataType.icon == null}"></i>
+							{{ dataType.name }}
+						</a>
+					</li>
+				</ul>
+			</div>
+		</div>
+
+		<h5 class="-border-bottom -black">Available editors</h5>
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web.UI.Client/src/views/common/overlays/contenttypeeditor/editorpicker/editorpicker.html]: Localize?

File: src/Umbraco.Web.UI.Client/src/views/common/overlays/contenttypeeditor/editorpicker/editorpicker.html (hunk 1 above)
Comment (madsrasmussen) [inline src/Umbraco.Web.UI.Client/src/views/common/overlays/contenttypeeditor/editorpicker/editorpicker.html]: argh damn.. will fix :)

**Comment by nul800sebastiaan** [comment]
@madsrasmussen Merged it in, I'll do the localization!

**Pull request merged**